registry/auth/token/accesscontroller.go

```
@@ -153,6 +162,8 @@ func checkOptions(options map[string]interface{}) (tokenAccessOptions, error) {
 
 	opts.realm, opts.issuer, opts.service, opts.rootCertBundle = vals[0], vals[1], vals[2], vals[3]
 
+	opts.autoRedirect, _ = options["autoredirect"].(bool)
```

Let's throw an error if it's not bool type